conjur/init/init_controller.py

```diff
@@ -58,6 +62,10 @@ def get_server_certificate(self):
                 # pylint: disable=raise-missing-from
                 raise RuntimeError("Error: URL is required")
 
+        # Chops off the '/ if supplied by the user to avoid a server error
+        if self.conjurrc_data.appliance_url.endswith('/'):
+            print(self.conjurrc_data.appliance_url[:-1])
```

Comment:
why do we print?

Reply:
great catch this shouldn't have made it to the PR